spec/services/idv/session_spec.rb

```diff
@@ -8,6 +8,44 @@
     Idv::Session.new(user_session: user_session, current_user: user, service_provider: nil)
   }
 
+  describe '#initialize' do
+    context 'without idv user session' do
+      it 'initializes user session' do
+        expect_any_instance_of(Idv::Session).to receive(:new_idv_session).twice.and_call_original
```

Comment:
Non-blocking but it seems unnecessary to me to dive into the implementation details to see which private methods are/aren't called when we can just check the session before and after to make sure the expected behavior occurs

Reply:
Yeah I think the tricky thing here is that with your suggestion, these specs would likely pass with the previous implementation? Since the goal is to make sure we're not assigning a `new_idv_session` if the session key already exists.